galaxy_ng/tests/integration/api/test_v3_plugin_paths.py

```diff
@@ -39,7 +39,7 @@ def test_api_v3_plugin_execution_environments_repositories_content_history(
     validate_json(instance=resp, schema=schema_objectlist)
 
     # assert on the expected minimum length of the dataset
-    assert resp['data'][0]['number'] >= 3
+    assert resp['data'][0]['number'] >= 1
```

Comment:
@jerabekjiri I think you originally wrote this test in https://github.com/ansible/galaxy_ng/commit/3b25dfcdfd441a1a59e8d62f92befef53cc9bb6f#diff-558ffa04db083a43eb6898222e0b8edf1e66b485e688c2bc66fd031917ee867d ? Do you know why this number might have gone from 3 to 1 with this upgrade?

Reply:
no idea tbh, I only moved these tests in the PR from `test_ui_paths.py` file. I think originally @bmclaughlin wrote them in https://github.com/ansible/galaxy_ng/pull/1462/files#diff-2db66d1f5ec6ec5f77814c403767648287f37dceac84f956a1b5757b7c2ec2f1R252.